src/unlimited.cpp

```diff
@@ -210,12 +209,12 @@ void HandleExpeditedRequest(CDataStream& vRecv,CNode* pfrom)
             if (xpeditedBlk.size() < maxExpedited )
 	      {
 		LogPrint("blk", "Starting expedited blocks to peer %s (%d).\n", pfrom->addrName.c_str(),pfrom->id);
-		std::vector<CNode*>::iterator it = std::find(xpeditedBlk.begin(), xpeditedBlk.end(),((CNode*)NULL));
+		std::vector<CNode*>::iterator it = std::find(xpeditedBlk.begin(), xpeditedBlk.end(),(CNode*)NULL);  // find an empty array location
```

Comment:
The code **was** changed (removal of parenthesis around `(CNode*)NULL` - not sure of the significance of that.